src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -72,7 +72,7 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
             boolean sessionIsOpen, int qnIdx, int responseIdx, String courseId, int totalNumRecipients) {
         return Templates.populateTemplate(
                 FormTemplates.TEXT_SUBMISSION_FORM,
-                Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                Slots.IS_SESSION_CLOSED, sessionIsOpen ? "false" : "true",
```

Comment:
Might want to change this line to the following code instead.

```Java
Slots.IS_SESSION_CLOSED, !sessionIsOpen ? "true" : "false",
```

This is again something I have seen across the system. And it does read better to have the first case in a Ternary seem like the logical outcome of a `true` evaluation of the condition. However, since this is something of a special case (as the outcomes are switched), we could ask @damithc to either weigh in himself or ask another senior dev to give their opinion.